WebClient/src/components/graph/graphViewControllerFactory.js

```diff
@@ -16,12 +16,11 @@ var GraphViewControllerFactory = L.Evented.extend({
         };
     },
 
-    create: function (graphModel, windowManager, graphViewOptions) {
+    create: function (graphModel, graphViewOptions) {
         var ViewController = this._viewControllerMap[graphModel.type] || GraphViewController;
```

Comment:
Maybe a good idea to add a comment here why the generic GraphViewController is or-ed with the look-upped type